scripts/resolve_markets.py

```diff
@@ -0,0 +1,51 @@
+from datetime import datetime, timezone
+
+import typer
+from eth_typing import HexAddress
+from eth_typing.evm import HexStr
+from prediction_market_agent_tooling.loggers import logger
+from prediction_market_agent_tooling.markets.omen.omen_subgraph_handler import (
+    OmenSubgraphHandler,
+)
+
+from prediction_market_agent.agents.ofvchallenger_agent.deploy import OFVChallengerAgent
+from prediction_market_agent.utils import APIKeys
+
+MARKET_IDS: list[HexAddress] = [
+    HexAddress(HexStr("000000000000000000000000000000000000000000")),
+]
+
+
+def main() -> None:
```

Comment:
You used `typer.run(main)` which is cool, that allows you to do:

```
def main(market_id: list[str]) -> None:
```

and then script can be used as

```
python scripts/resolve_markets.py id-1 id-2 ...
```

so no hard-coding of market ids is necessary. But you will need to convert them to proper type using

```
from web3 import Web3
Web3.to_checksum_address(addr)
```